lib/menu.rb

```diff
@@ -0,0 +1,18 @@
+class Menu
+
+
+  attr_reader :foods
+
+  def initialize
+    @foods = { "pizza" => 7.99,
+              "pasta" => 7.50,
+              "steak" => 11.99,
+              "pint" => 3.99 }
+  end
```

Comment:
look at injecting the foods in rather than setting the instance variable like this. At the moment @foods can only be the above. You could a contant in the call as DEFAULT_FOODS